packages/radix-ui-themes/src/components/table.css

```diff
@@ -32,16 +40,17 @@
   box-shadow: var(--table-row-box-shadow);
   box-sizing: border-box;
   vertical-align: inherit;
-  padding: var(--table-cell-padding);
+  padding: var(--table-cell-padding-top) var(--table-cell-padding-right) var(--table-cell-padding-bottom)
+    var(--table-cell-padding-left);
   /* Works as min-height */
   height: var(--table-cell-min-height);
 
   /* Inset with Table */
   .rt-Inset :where(&:first-child) {
-    padding-left: var(--inset-padding-left, var(--table-cell-padding));
+    padding-left: var(--inset-padding-left, var(--table-cell-padding-left));
```

Comment:
This fixes a potential (but unlikely) bug, where the fallback value would be invalid in cases where `--table-cell-padding` evaluates to a shorthand value.